catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -273,6 +274,28 @@ protected void correctJdbcTableFields(
     }
   }
 
+  protected void getTableStatus(Connection connection, String databaseName, String tableName) {
+    LOG.info("=======================");
+    // sql is `SHOW ALTER TABLE COLUMN`
+    String sql = String.format("SHOW ALTER TABLE COLUMN WHERE TableName = '%s'", tableName);
```

Comment:
Why doesn't the table have column database names? I'm very confused about it. Table names are not unique in some cases.

Reply:
Because `SHOW ALTER TABLE COLUMN` only can be executed under database.  

If database info is not setted in connection, we will get an excepiton.